clang/lib/Headers/hlsl/hlsl_intrinsic_helpers.h

```diff
@@ -10,7 +10,7 @@
 #define _HLSL_HLSL_INTRINSIC_HELPERS_H_
 
 namespace hlsl {
-namespace __detail {
+namespace __dETAil {
```

Comment:
this looks like you did find replace for eta. Please revert this change. In the future you can use clangd' vscode plugin to apply llvm c++'s stye guide.